src/extension/background-script/actions/liquid/__tests__/signPset.test.ts

```diff
@@ -96,8 +96,6 @@ describe("decode PSET", () => {
         expect(inputs.at(0)?.amount).toBe(1_0000_0000);
         expect(inputs.at(0)?.asset).toBe(networks.regtest.assetHash);
         expect(outputs.length).toBeGreaterThan(0);
-      } else {
```

Comment:
Note: I think the `else` case could be handled or maybe invalid values should be filtered out of `liquidFixtureDecode`